### PR TITLE
Updating splash about

### PR DESCRIPTION
**Pull request by PhillypHenning**
Quick update to splash text

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -15,8 +15,7 @@ hide:
 
 <div class="bitovi-row">
     <div class="bitovi-column">
-        <h1>Organize your Infrastructure-as-Code (IAC) 
-        <nl>Consistently Deploy.</h1>
+        <h1>BitOps packages, centralizes and organizes IaC</h1>
```
Comment (armab) [inline docs/index.md:18]: Would that work?
```suggestion
        <h1>BitOps packages, centralizes and organizes your Infrastructure-as-Code</h1>
```
I think it's nice to spell the IaC explicitly as we're targeting the entry group where the `IaC` abbreviation could be not that clear.

**Review by armab — APPROVED** [review]
Looks much more clear to me 👍

**Comment by armab** [comment]
@mickmcgrath13 @ccapell WDYT? ^^

We went to the docs and it looked confusing:
image
Now I see that `Consistently Deploy` is a separated `<h1>`, though the new version by @PhillypHenning looks like a better description.

**Review by ccapell — APPROVED** [review]
Looks good to me.  

@armab, you won't see the changes in the documentation site until they are approved and pushed.

File: docs/index.md [hunk 2]
```diff
@@ -15,8 +15,7 @@ hide:
 
 <div class="bitovi-row">
     <div class="bitovi-column">
-        <h1>Organize your Infrastructure-as-Code (IAC) 
-        <nl>Consistently Deploy.</h1>
+        <h1>BitOps packages, centralizes and organizes your Infrastructure-as-Code</h1>
```
Comment (mickmcgrath13) [inline docs/index.md:18]: Does BitOps package your IaC?  I think of "packaging" as "building artifacts for deployment", and neither bitops nor the ops repo does that for your IaC.

What about just
> BitOps centralizes and organizes your Infrastructure-asCode

?

Comment (armab) [reply docs/index.md:18]: How about replacing "packages" -> "deploys"?

```suggestion
        <h1>BitOps centralizes, organizes and deploys your Infrastructure-as-Code</h1>
```
In the past version there was mention of deployments.

Comment (mickmcgrath13) [reply docs/index.md:18]: `deploys` is good.  I recommend adding the Oxford comma ;). @PhillypHenning

File: docs/index.md [hunk 3]
```diff
@@ -15,8 +15,7 @@ hide:
 
 <div class="bitovi-row">
     <div class="bitovi-column">
-        <h1>Organize your Infrastructure-as-Code (IAC) 
-        <nl>Consistently Deploy.</h1>
```
Comment (armab) [inline docs/index.md:19]: lol, just noticed we had `<n1>` tag here :)

**Comment by armab** [comment]
Looks like it's ready to merge @PhillypHenning with the consensus 👍

**Review by mickmcgrath13 — APPROVED** [review]

**Pull request merged**